api/controllers/expenseControllers.js

```diff
@@ -0,0 +1,104 @@
+const Expense = require('../models/expense');
+
+const SERVER_STATUS_ERROR = 500;
+const USER_STATUS_ERROR = 422;
+
+const expenseCreate = (req, res) => {
+  const { amount, description, budget, categories } = req.body;
+  const newExpense = new Expense({ amount, description, budget, categories });
+  newExpense
+    .save()
+    .then(expense => {
+      if (expense === null) {
```

Comment:
I don't think this should be a 500 err. I think it's more an  422 in that if the thing isn't there, then the user requested an 'Unprocessable entity'. I hope that makes senes? It's a user error in the sense that the user requested something that wasn't there...